src/app/ClusterStateCache.h

```diff
@@ -86,8 +86,10 @@ class ClusterStateCache : protected ReadClient::Callback
         virtual void OnEndpointAdded(ClusterStateCache * cache, EndpointId endpointId){};
     };
 
-    ClusterStateCache(Callback & callback, Optional<EventNumber> highestReceivedEventNumber = Optional<EventNumber>::Missing()) :
-        mCallback(callback), mBufferedReader(*this)
+    ClusterStateCache(Callback & callback, bool cacheData = true,
```

Comment:
This should document what passing `cacheData = false` would mean.

Reply:
done, thanks